packages/cli/generators/relation/has-many-relation.generator.js

```diff
@@ -44,9 +44,8 @@ module.exports = class HasManyRelationGenerator extends BaseRelationGenerator {
     this.artifactInfo.targetModelRequestBody = utils.camelCase(
       this.artifactInfo.targetModelName,
     );
-    this.artifactInfo.relationPropertyName = utils.pluralize(
-      utils.camelCase(options.destinationModel),
-    );
+    this.artifactInfo.relationPropertyName = options.relationName;
```

Comment:
is `utils.pluralize` and `utils.camelCase` still needed?

Reply:
Yes, we need them for path names and model/repo names.

Reply:
oh 🙈 then why they are removed in the new code?

Reply:
Sorry I thought you meant the import declaration. 

This is how we defined **default relation name** before. But it didn't respect the relation name we got from the prompt:
```
    this.artifactInfo.relationPropertyName = utils.pluralize(
      utils.camelCase(options.destinationModel),
    );
```
And the new change would allow the customized relation name.

Reply:
oh I see. 👍